test/AssetPool.test.js

```diff
@@ -910,6 +922,51 @@ describe("AssetPool", () => {
     })
   })
 
+  describe("totalValue", () => {
+    context("when there is nothing in the pool", () => {
+      it("should return zero", async () => {
+        expect(await assetPool.totalValue()).to.equal(to1e18(0))
```

Comment:
I think we can assert `to.equal(0)`

Reply:
https://github.com/keep-network/coverage-pools/pull/121/commits/9310e0287d0390342f90f04143ae6a3ee215767c